actors/eoa/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+use fvm_ipld_encoding::{Cbor, RawBytes};
+use fvm_shared::{MethodNum, METHOD_CONSTRUCTOR};
+use num_derive::FromPrimitive;
+use num_traits::FromPrimitive;
+use serde::Deserialize;
+use serde::Serialize;
+
+use fil_actors_runtime::runtime::{ActorCode, Runtime};
+use fil_actors_runtime::{actor_error, restrict_internal_api, ActorError, SYSTEM_ACTOR_ADDR};
+
+#[cfg(feature = "fil-actor")]
+fil_actors_runtime::wasm_trampoline!(Actor);
+
+/// Externally Owned Address actor methods.
+#[derive(FromPrimitive)]
+#[repr(u64)]
+pub enum Method {
+    Constructor = METHOD_CONSTRUCTOR,
+}
+
+/// Externally Owned Address actor state.
+#[derive(Default, Deserialize, Serialize)]
+#[serde(transparent)]
+pub struct State([(); 0]);
+impl Cbor for State {}
+
+/// Externally Owned Address actor.
+pub struct Actor;
+impl Actor {
+    /// Externally Owned Address actor constructor.
+    pub fn constructor(rt: &mut impl Runtime) -> Result<(), ActorError> {
+        rt.validate_immediate_caller_is(std::iter::once(&SYSTEM_ACTOR_ADDR))?;
+
+        rt.create(&State::default())?;
+        Ok(())
+    }
+}
+
+impl ActorCode for Actor {
+    fn invoke_method<RT>(
+        rt: &mut RT,
+        method: MethodNum,
+        _params: &RawBytes,
+    ) -> Result<RawBytes, ActorError>
+    where
+        RT: Runtime,
+    {
+        restrict_internal_api(rt, method)?;
```

Comment:
How does this work in the presence of the EVM?

Reply:
I.e.: The EVM is both a "builtin actor" and the user's agent.

Reply:
@Stebalien It...didn't. Fixed here https://github.com/filecoin-project/builtin-actors/pull/932.